federation/cmd/federation-apiserver/app/server.go

```diff
@@ -196,6 +197,8 @@ func Run(s *options.ServerRunOptions) error {
 		return err
 	}
 
+	routes.Version{}.Install(m.Mux, m.HandlerContainer)
```

Comment:
It's kube specific at least as it depends on the Kubernetes version package. We could move the whole handler into the version package. How does that sound? Another alternative would be to abstract the version getter into an interface and inject that via the config.